import.py

```diff
@@ -0,0 +1,281 @@
+    print("Skipping simplification because this file has previously been reprojected.")
+  else:
+    print("Simplifying with tolerance", tolerance)
+    ogrCmd = [
+      "ogr2ogr",
+      simplified,
+      original,
+      "-simplify", tolerance
+    ]
+    subprocess.call(ogrCmd)
+    print(original, "simplified.")
+  return simplified
+
+
+
+def askUserForFieldNames(sf, stem):
+  fieldNames = [x[0] for x in sf.fields[1:]]
+  print("Found the following fields in the attribute table:")
+  print(str(fieldNames).strip("[").strip("]").replace("'",""))
+  print("Which would you like to use to look up snuggets by?")
+  keyField = False
+  while keyField not in fieldNames:
+    keyField = input(">> ")
+#TODO: make sure it really is OK for key & unique fields to be the same
+  print("Which is unique for each shape? (can be the same field as before)")
+  uniqueField = False
+  while uniqueField not in fieldNames:
+    uniqueField = input(">> ")
+  print("Generating code for", stem, "using the following fields:")
+  print("'" + keyField + "' to look up snuggets.")
+  print("'" + uniqueField + "' as the unique ID.")
+  return keyField, uniqueField
+
+
+
+def detectGeometryType(sf, stem):
+  shapeType = 0
+  i = 0
+  while shapeType == 0:
+    shapeType = sf.shapes()[i].shapeType
+    i = i + 1
+  if shapeType == 5:
+    return "MultiPolygon"
+  elif shapeType == 3:
+  	print("WARNING:", stem, "has a linear geometry, and this application currently only handles polygons properly")
+  	return "MultiLineString"
+  else:
+  	print("Geometry field type ", shapeType, "unrecognised")
+  	# the list of valid geometry field type codes is at
+  	# https://www.esri.com/library/whitepapers/pdfs/shapefile.pdf p4
+  	# but see also caveat at
+  	# https://gis.stackexchange.com/questions/122816/shapefiles-polygon-type-is-it-in-fact-multipolygon
+  	exit()
+
+
+
+def findEncoding(sf, inputDir, stem):
+  encodingFile = os.path.join(inputDir, stem+".cpg")
+  if os.path.exists(encodingFile):
+    with open(encodingFile, 'r') as f:
+      encoding = f.read()
+    print("Determined that", stem, "uses character encoding", encoding)
+# TODO: implement the chardet method from https://gist.github.com/jatorre/939830 as another option
+  else:
+    print("Unable to automatically detect the character encoding of", stem)
+    print("What encoding should we use? (If in doubt, choose UTF-8)")
+    encoding = input(">> ")
+  return encoding
+
+
+
+def findFieldType(sf, fieldName):
+  for field in sf.fields:
+    if field[0] == fieldName:
+      if field[1] == 'C':
+        return "CharField(max_length=" + str(field[2]) + ")"
+      elif field[1] == 'N':
+        if field[3] > 0:
+          return "FloatField()"
+        else:
+          return "IntegerField()"
+      else:
+        print("Field type unrecognised:")
+        print(field)
+        exit()
+
+
+
+def modelClassGen(stem, sf, keyField, uniqueField, srs, shapeType):
+  text = "class " + stem + "(models.Model):\n"
```

Comment:
since you did it twice, consider assigning something at the top to keyField.lower() and using that.